### PR TITLE
uuid added

### PR DESCRIPTION
**Pull request by vovuk097**

### CODE REVIEW

File: db/migrate/20161206204443_create_orders_items.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateOrdersItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :orders_items, id: :uuid do |t|
+      t.integer :order_id, index: true
```
Comment (ruslan-369) [inline db/migrate/20161206204443_create_orders_items.rb]: uuid

File: db/migrate/20161206204443_create_orders_items.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      t.integer :item_id, index: true
```
Comment (ruslan-369) [inline db/migrate/20161206204443_create_orders_items.rb]: uuid

File: db/migrate/20161206204443_create_orders_items.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+      t.timestamps
```
Comment (ruslan-369) [inline db/migrate/20161206204443_create_orders_items.rb]: null: false

**Review by ruslan-369 — CHANGES_REQUESTED** [review]

File: db/migrate/20161206204443_create_orders_items.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateOrdersItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :orders_items, id: :uuid do |t|
+      t.uuid :order_id, index: true
+      t.uuid :item_id, index: true
```
Comment (nickKolya) [inline db/migrate/20161206204443_create_orders_items.rb:5]: could u remove `order_id` from item?

Comment (nickKolya) [reply db/migrate/20161206204443_create_orders_items.rb:5]: it's unnecessary anymore and also change relation

**Review by nickKolya — CHANGES_REQUESTED** [review]

File: db/migrate/20161204174750_devise_create_users.rb [hunk 5]
```diff
@@ -1,6 +1,7 @@
 class DeviseCreateUsers < ActiveRecord::Migration[5.0]
   def change
-    create_table :users do |t|
+    enable_extension 'uuid-ossp'
```
Comment (ruslan-369) [inline db/migrate/20161204174750_devise_create_users.rb:3]: better to add to different file named like 'add_uuid_extension'

**Review by ruslan-369 — DISMISSED** [review]

**Review by nickKolya — APPROVED** [review]

**Pull request merged**